lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -3368,6 +3376,26 @@ public function buildColumnUnsignedProvider() {
 					'default' => 1
 				),
 				'`testName` decimal UNSIGNED DEFAULT 1'
+			),
+			//set #9
```

Comment:
But it has also some disadvantage. Even if people uses the same key, PHP doesn't complain it. Some important data might be not provided in such cases. If our code sniffer cannot complain it, I am not in favor of that.